docs/user_guide/mitigation.rst

```diff
@@ -194,6 +207,684 @@ Postprocessing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Using :class:`RejectOptionClassifier` requires manually specifying the critical region, the
+group to (up/down) select, and the selection label.
+
+As mentioned above, :class:`RejectOptionClassifier` assumes that the sensitive feature is binary.
+Of course, it can happen that there are multiple sensitive features, or that the sensitive feature
+is not binary (just like in our previous example). In this case it is easiest to create a new
+sensitive feature column which is binary. To see how this, and using
+:class:`RejectOptionClassifier` itself, works, consider the same 
+:class:`sklearn.ensemble.RandomForestClassifier` and :ref:`data <hospital_readmissions_dataset>`
+as in the section on :ref:`Thresholder<default_predictions>`.
+
+.. doctest:: mitigation
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> #create binary sensitive feature vector 
+    >>> sf_train_reject_clf = sf_train.copy()\
+    ...                        .replace(['Asian','Other'], 'upselect')\
+    ...                        .replace(['Caucasian','AfricanAmerican','Hispanic'], 'downselect')
+
+    >>> sf_test_reject_clf = sf_test.copy()\
+    ...                        .replace(['Asian','Other'], 'upselect')\
+    ...                        .replace(['Caucasian','AfricanAmerican','Hispanic'], 'downselect')
+
+    >>> #Investigate predictions before using RejectOptionClassifier
+    >>> MetricFrame(metrics=false_negative_rate, 
+    ...             y_true=Y_test,
+    ...             y_pred=Y_pred_clf, 
+    ...             sensitive_features=sf_test_reject_clf).by_group 
+    race
+    downselect    0.397810
+    upselect      0.380952
+    Name: false_negative_rate, dtype: float64
+
+    >>> plot_positive_predictions(sf_test_reject_clf,Y_pred_clf)
+
+.. figure:: ../auto_examples/images/user_guide_reject_option_clf_binary_output_before.png
+    :align: center
+
+
+.. doctest:: mitigation
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> #import and use RejectOptionClassifier for predictions
+    >>> from fairlearn.postprocessing import RejectOptionClassifier
+
+    >>> reject_clf = RejectOptionClassifier(estimator=classifier,
+    ...                                     critical_width=0.2,
+    ...                                     group_to_upselect='upselect',
+    ...                                     group_to_downselect='downselect',
+    ...                                     selection_label=1,
+    ...                                     prefit=True)\
+    ...                                     .fit(X_train,Y_train,sensitive_features=sf_train_reject_clf)
+
+    >>> Y_pred_reject_clf = reject_clf.predict(X_test,sensitive_features=sf_test_reject_clf)
+
+    >>> MetricFrame(metrics=false_negative_rate, 
+    ...             y_true=Y_test,
+    ...             y_pred=Y_pred_reject_clf, 
+    ...             sensitive_features=sf_test_reject_clf).by_group
+    race
```

Comment:
If you're using a contrived example then it's probably best not to call it "race"